src/main/java/com/taf/core/ConfigManager.java

```diff
@@ -2,16 +2,18 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.taf.business.exceptions.ProcessingException;
-import com.taf.core.dto.User;
+import com.taf.core.models.Admin;
+import com.taf.core.models.User;
 
 import java.io.FileInputStream;
 import java.util.Properties;
 
 public class ConfigManager {
```

Comment:
to be consistent with the logic of the class it should be renamed to `UserConfigManager`

Reply:
Fixed after code refactoring.